src/component/Home.js

```diff
@@ -7,7 +7,7 @@ function Home({ token, setToken }) {
     <div className="homeWrap">
       <h1>New Shopping List</h1>
       <GetToken token={token} setToken={setToken} />
-      <Link to="/join" className="button-link">
+      <Link to="/join" className="button-link" id="join-list">
```

Comment:
There are a few changes in this PR that don't appear on first glance to be related to the AC. Not blocking merging the PR as it is, I'm just curious what necessitated these changes related to joining a list?

Reply:
Yes! We noticed a few failing tests and fixed them - I think during a previous merge to master, some of the tests broke and no one noticed. It probably would've been better to start a new branch and PR'd that when we noticed them and then merge those changes into our branch - is that right?

Reply:
That's correct. Ideally, we'd make a PR to get the code in good shape so that this PR could be focused just on the changes needed to complete the AC. All good, though, was just curious!